pkgs/servers/meguca/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, go, nodejs, pkgconfig, ffmpeg, graphicsmagick, ghostscript }:
+
+stdenv.mkDerivation rec {
+  name = "meguca-unstable-2018-04-25";
+
+  src = fetchgit {
+    url = "https://github.com/bakape/meguca.git";
+    rev = "4961e76743fc22c002fcf8f1bc8f77bbe73b2623";
+    sha256 = "055m323s5vki6dg77mm8sfj88ymh46bjckx8w474gyx5q59jprgp";
+    fetchSubmodules = false;
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ go nodejs ffmpeg graphicsmagick ghostscript ];
+
+  preBuild = ''
+    mkdir -p $PWD/.home/go
+    export HOME=$PWD/.home
+    export GOPATH=$HOME/go
+    export GIT_SSL_CAINFO=/etc/ssl/certs/ca-certificates.crt
+    export SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
I'd like especially if someone could tell me a better way to do this.
I'm not familiar with buildGoPackage and couldn't find a way to make it work with the way meguca's build flow goes.